<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectWizard</class>
 <widget class="QWizard" name="NewProjectWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="projectNameEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Project Name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Project File:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="projectFileEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getPathButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="templateCombo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Template:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="descriptionEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>NewProjectWizard</sender>
   <signal>accepted()</signal>
   <receiver>NewProjectWizard</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>getPathButton</sender>
   <signal>clicked()</signal>
   <receiver>NewProjectWizard</receiver>
   <slot>findProjectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findProjectFile()</slot>
 </slots>
</ui>
